tests/playwright/utils/po-manager.js

```diff
@@ -0,0 +1,20 @@
+const { GlobalUtils } = require( './globalUtils' );
+const { LibraryKits } = require( '../sanity/core/app/modules/kit-library/libraryKits.utils' );
+
+class POManager {
```

Comment:
class name not clear 
what is a PO ?

Reply:
Got rid of the PO Manager after talking to Hein